config/services/accounts.yml

```diff
@@ -2,6 +2,18 @@ services:
   _defaults:
     public: true
 
+  mbo.ps_accounts.installer:
+    class: 'PrestaShop\PsAccountsInstaller\Installer\Installer'
+    public: true
+    arguments:
+      - "5.2.5"
```

Comment:
Maybe we can put the 6.X version here as we are on a PS > 8.

Reply:
We need a task to upgrade it separately, will be probably included in the next major (?)